modules/rosa-create-objects.adoc

```diff
@@ -546,4 +597,53 @@ $ rosa create ocm-role [flags]
 
 |===
 
-For more information about the ocm-roles and the roles created with the `rosa create ocm-roles`, see the Additional resources section.
\ No newline at end of file
+For more information about the OCM role created with the `rosa create ocm-role` command, see _Account-wide IAM role and policy reference_.
+
+[id="rosa-create-user-role_{context}"]
+== create user-role
+
+Create the needed user-role resources for your cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create user-role [flags]
+----
+
+.Flags
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--debug
+|Enable debug mode.
+
+|-i, --interactive
+|Enable interactive mode.
+
+|-m, --mode string
+|How to perform the operation. Valid options are:
```

Comment:
Again I know this is from existing content, but this is hard to read when it's rendered. Recommend making it actual bullets or something for better readability, and also putting `auto` and `manual` in backticks

Reply:
Good point. Resolved.